homeassistant/components/tuya/select.py

```diff
@@ -27,6 +27,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        SelectEntityDescription(
+            key=DPCode.DEVICE_MODE,
+            name="PIR mode",
+            icon="mdi:arrow-decision-auto",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.CDS,
```

Comment:
What would these provide for your device?

Reply:
The CDS level is predefined by Tuya to various lux values "2000, 300, 50, 10, 5, now" (reflecting Day, after dusk, after evening, after night and dark) . It defines the maximum ambient light levels the PIR will trigger turn on the light and trigger the Motion event when motion is detected.

Reply:
Translation facility added